ocrd_utils/ocrd_utils/str.py

```diff
@@ -62,24 +62,25 @@ def make_file_id(ocrd_file, output_file_grp):
     Derive a new file ID for an output file from an existing input file ``ocrd_file``
     and the name of the output file's ``fileGrp/@USE``, ``output_file_grp``.
     If ``ocrd_file``'s ID contains the input file's fileGrp name, then replace it by ``output_file_grp``.
+    Else if ``ocrd_file``'s ID contains the input file's pageId, then merely append ``output_file_grp``.
     Otherwise use ``output_file_grp`` together with the position of ``ocrd_file`` within the input fileGrp
-    (as a fallback counter). Increment counter until there is no more ID conflict.
+    (as a fallback counter), and increment counter until there is no more ID conflict.
     """
     ret = ocrd_file.ID.replace(ocrd_file.fileGrp, output_file_grp)
     if ret == ocrd_file.ID:
-        m = re.match(r'.*?(\d{3,}).*', ocrd_file.pageId or '')
-        if m:
-            n = int(m.group(1))
+        if ocrd_file.pageId and ocrd_file.pageId in ocrd_file.ID:
+            # still sufficiently unique
+            ret = ocrd_file.ID + '_' + output_file_grp
```

Comment:
It doesn't matter wrt to uniqueness but would it not be more consistent to do

```python
ret = output_file_grp + '_' + ocrd_file.ID
```

to imitate the OCR-D-internal convention for file IDs?

Reply:
Right! Sry, I was too hasty there.

(Edited you comment as a suggestion and commited just now.)